inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/pulsar/src/main/java/org/apache/inlong/sort/pulsar/PulsarTableOptions.java

```diff
@@ -282,4 +282,11 @@ private PulsarTableOptions() {
                     .booleanType()
                     .defaultValue(true)
                     .withDescription("Indicate if the table is an explicit Flink table.");
+
+    public static final ConfigOption<Boolean> GENERIC = ConfigOptions
```

Comment:
What the meaning if false?Maybe it is needed a clearer definition.

Reply:
there is no use of this parameter in flink 1.15 , so I removed it 
for flink1.13 its value is always true, so I removed it in flink 1.13 also to keep consistent